tpmutil/encoding.go

```diff
@@ -51,6 +51,12 @@ func UseTPM20LengthPrefixSize() {
 func packedSize(elts ...interface{}) (int, error) {
 	var size int
 	for _, e := range elts {
+		// Check if it's a SelfMarshaler
```

Comment:
No need for this comment, since it repeats exactly what the code is already doing.

Please also remove this comment from the other places this happens below.